tests/fullstack/fullstack.go

```diff
@@ -19,13 +19,15 @@ import (
 	systemimpl "github.com/textileio/go-tableland/internal/system/impl"
```

Comment:
Nothing much interesting here. I had to move around some wires since the user store now depends on the event processors for creating resolvers. Mostly boilerplate stuff.